packages/ckeditor5-table/tests/manual/table.html

```diff
@@ -201,4 +201,38 @@
 		</tr>
 		</tbody>
 	</table>
+
+	<p>Nested tables</p>
```

Comment:
The same change would be good in the `tableproperties.html` manual test. Table props is a feature that I'd be worried may not work well with nested tables.

Reply:
Nested and styled table has been added to `tableproperties.html` test.